go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -193,16 +193,24 @@ func runOvnKube(ctx *cli.Context) error {
 		return err
 	}
 
+	master := ctx.String("init-master")
+	node := ctx.String("init-node")
+
 	// create factory and start the controllers asked for
+	var wf *factory.WatchFactory
 	stopChan := make(chan struct{})
-	factory, err := factory.NewWatchFactory(clientset, stopChan)
-	if err != nil {
-		return err
+	if master == "" && node != "" {
+		wf, err = factory.NewNodeWatchFactory(node, clientset, stopChan)
+		if err != nil {
+			return err
+		}
+	} else {
+		wf, err = factory.NewWatchFactory(clientset, stopChan)
```

Comment:
`NewMasterWatchFactory`?

Reply:
Yes, though the code still supports combined master + node mode. So this is the Everything watch factory.